__init__.py

```diff
@@ -73,6 +73,7 @@ def __init__(self):
 
         self.idle_screens = {}
         self.override_idle = None
+        self.override_animations = None
```

Comment:
Should probably be initialized to False (since it can really only be True or False)